example-test-gitlab4j.properties

```diff
@@ -1,5 +1,33 @@
-# Rename this file to test-gitlab4j.properties with proper property values.
+# To test GitLab4J-API you must have access to a gitlab server.
+#
+# Copy this file to test-gitlab4j.properties and update the property
+# values to match the gitlab server you are testing against.
+#
+# The TEST_PRIVATE_TOKEN and TEST_USERNAME should belong to an admin user
+#
+
+# The following values must be set or testing will fail
 TEST_NAMESPACE=some-namespace
 TEST_PROJECT_NAME=some-project-name
 TEST_HOST_URL=some-gitlab-url
-TEST_PRIVATE_TOKEN=some-private-token
+TEST_PRIVATE_TOKEN=admin-private-token
+TEST_ACCESS_TOKEN=some-access-token
+TEST_USERNAME=admin-user
+
+# To test the GroupApi, set these to an existing group and group-project, and group member
+TEST_GROUP=
+TEST_GROUP_PROJECT=
+TEST_GROUP_MEMBER_USERNAME=
+
+# To test oauth2Login, set these properties
+TEST_LOGIN_USERNAME=
+TEST_LOGIN_PASSWORD=
+
+# To test sudo capability provide a username to sudo as
+TEST_SUDO_AS_USERNAME=
+
+# To test using GitLab4J-API with a proxy, set the following properties
+TEST_PROXY_URI=
+TEST_PROXY_USERNAME=
+TEST_PROXY_PASSWORD=
```

Comment:
Should we follow the above convention of having some stub string such as `some-proxy-password`?

Reply:
As you imply, to be consistent it should be one way or the other.  Providing a value could lead to some confusion, (are these valid values?), so I suggest we remove all values, something like this:
```
#
# To test GitLab4J-API, you must have access to a GitLab server.
#
# Copy this file to test-gitlab4j.properties and set the property
# values to match the GitLab server you are testing against.
#

# REQUIRED: The following values must be set or the tests will fail.
# The TEST_PRIVATE_TOKEN and TEST_USERNAME must belong to an admin user. 
TEST_PRIVATE_TOKEN=
TEST_ACCESS_TOKEN=
TEST_USERNAME=
TEST_NAMESPACE=
TEST_PROJECT_NAME=
TEST_HOST_URL=

# OPTIONAL: To test the GroupApi, set these to an existing group name and group-project name,
# and the username of a group member
TEST_GROUP=
TEST_GROUP_PROJECT=
TEST_GROUP_MEMBER_USERNAME=

# OPTIONAL: To test oauth2Login, set these properties
TEST_LOGIN_USERNAME=
TEST_LOGIN_PASSWORD=

# OPTIONAL: To test sudo capability provide a username to sudo as
TEST_SUDO_AS_USERNAME=

# OPTIONAL: To test using GitLab4J-API with a proxy, set the following properties
TEST_PROXY_URI=
TEST_PROXY_USERNAME=
TEST_PROXY_PASSWORD=

```

Reply:
I agree with what you said about validity. My comment was more so on picking one convention over another.

Reply:
I figured that, I just pushed this change.